src/index.js

```diff
@@ -1 +1,24 @@
-import './style.css';
\ No newline at end of file
+import './style.css';
+import sendScores from './sendScore.js';
+import {
+  getScore, displayScores,
+} from './displayScore.js';
+
+const url = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/BT0dBrz8PnCOOMBT0tZ4/scores/';
+const form = document.getElementById('gameForm');
+const refreshBtn = document.getElementById('refreshBtn');
```

Comment:
- Your refreshBtn it's missing functionallity when you hit the ` GET API ` endpoint causing the scores list to change.

Reply:
Thanks for the review. But I add the functionality of the refresh button in `index.js`.